tabs/tab_logit.py

```
@@ -303,33 +314,212 @@ def ui_dataset_selector():
+                )
+            )
+        
+        return ui.div(
+            ui.div(*items, class_="d-flex flex-wrap"),
+            ui.div(
+                ui.input_action_button("btn_clear_interactions", "Clear All", class_="btn-xs btn-outline-danger mt-2"),
+            )
+        )
+        
+    @reactive.Effect
+    def _handle_interaction_removal():
```

Ruff (0.14.10)

Missing return type annotation for private function `_handle_interaction_removal`

Add return type annotation: `None`

(ANN202)